polylogue/cli/init.py

```
@@ -97,7 +111,38 @@ def run_init_cli(args: argparse.Namespace, env: CommandEnv) -> None:
+    qdrant_vector: Optional[int] = CONFIG.index.qdrant_vector_size if CONFIG.index else None
+    if not ui.plain:
+        backend_choice = ui.choose("  Backend (sqlite/qdrant/none)", ["sqlite", "qdrant", "none"])
+        backend = backend_choice or "sqlite"
+        if backend == "qdrant":
+            qdrant_url = ui.input("  Qdrant URL", default=qdrant_url or "http://localhost:6333")
+            qdrant_key = ui.input("  Qdrant API key (blank for none)", default=qdrant_key or "")
+            qdrant_collection = ui.input("  Qdrant collection", default=qdrant_collection or "polylogue")
+            vector_raw = ui.input("  Vector size", default=str(qdrant_vector or 1536))
+            try:
+                qdrant_vector = int(vector_raw)
+            except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)